android/src/main/kotlin/com/airship/flutter/AirshipPlugin.kt

```diff
@@ -110,7 +116,12 @@ class FlutterInboxMessageView(private var context: Context, channel: MethodChann
     }
 }
 
-class AirshipPlugin : MethodCallHandler {
+class AirshipPlugin : MethodCallHandler, FlutterPlugin, ActivityAware, ServiceAware {
```

Comment:
do we need to implement ActivityAware and ServiceAware interfaces?

Reply:
In the Flutter docs, it says :
```
(Optional) If your plugin needs an Activity reference, also implement the ActivityAware interface.

(Optional) If your plugin is expected to be held in a background Service at any point in time, implement the ServiceAware interface.
```

And if I'm not wrong, I guess the Plugin interacts with the Activity, and is expected to be held in a background Service.
However, looks like everything is working fine even when I remove ActivityAware and ServiceAware. So I guess I will remove them